[2021-12-22 17:35:21,742] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.sketch scheduled__2021-12-20T00:22:00+00:00 [queued]>
[2021-12-22 17:35:21,745] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.sketch scheduled__2021-12-20T00:22:00+00:00 [queued]>
[2021-12-22 17:35:21,746] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 17:35:21,746] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-22 17:35:21,746] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 17:35:21,755] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): sketch> on 2021-12-20 00:22:00+00:00
[2021-12-22 17:35:21,758] {standard_task_runner.py:52} INFO - Started process 28852 to run task
[2021-12-22 17:35:21,762] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline1', 'sketch', 'scheduled__2021-12-20T00:22:00+00:00', '--job-id', '139', '--raw', '--subdir', '/home/myamya/project/dockers/airflow/dags/hello_world.py', '--cfg-path', '/tmp/tmp1fbdt6zo', '--error-file', '/tmp/tmptsqtv33j']
[2021-12-22 17:35:21,763] {standard_task_runner.py:77} INFO - Job 139: Subtask sketch
[2021-12-22 17:35:21,781] {logging_mixin.py:109} INFO - Running <TaskInstance: data_pipeline1.sketch scheduled__2021-12-20T00:22:00+00:00 [running]> on host myamya
[2021-12-22 17:35:21,802] {taskinstance.py:1412} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yangjae
AIRFLOW_CTX_DAG_ID=data_pipeline1
AIRFLOW_CTX_TASK_ID=sketch
AIRFLOW_CTX_EXECUTION_DATE=2021-12-20T00:22:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-20T00:22:00+00:00
[2021-12-22 17:35:21,802] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-22 17:35:21,803] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'docker run -it -v /home/myamya/project/image_files/:/home/myamya/project/image_files/ sketch:0.4 /bin/bash /home/myamya/project/init.sh ']
[2021-12-22 17:35:21,806] {subprocess.py:85} INFO - Output:
[2021-12-22 17:35:42,320] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-22 17:35:42,352] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=data_pipeline1, task_id=sketch, execution_date=20211220T002200, start_date=20211222T083521, end_date=20211222T083542
[2021-12-22 17:35:42,405] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-22 17:35:42,422] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
